### PR TITLE
[Staging] Split viz types into components with defined style

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

For current visualization display, split each viz to different components by type and allow the styles to take full width.
cc @kilemensi 

Fixes # (issue)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Screenshots

Screen Shot 2019-04-29 at 10 57 00 AM

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
> QQ @karimkawambwa:
> 
> Why are widget components still under `FeaturedData`? I'm thinking they'll be stand-alone components that may be used from any other component, `FeaturedData`, `AnalysisPageContent`, etc.,

The design I went with, you should pass a type variable in your data using the DataContainer.

**Comment by karimkawambwa** [comment]
@kilemensi 👀 on refactor

File: takwimu/takwimu_ui/src/components/DataContainer/EntitiesDataContainer.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+};
+
+function DataContainer({ id, classes, data }) {
+  const [images, setImages] = useState({});
+
+  useEffect(() => {
+    data.entities.forEach(
+      entity =>
+        entity.image &&
+        fetch(`/api/v2/images/${entity.image}`)
```
Comment (kilemensi) [inline takwimu/takwimu_ui/src/components/DataContainer/EntitiesDataContainer.js]: What is happening with this fetch? Aren't URLs returned as part of the API i.e. https://docs.wagtail.io/en/v2.3/advanced_topics/api/v2/configuration.html#images-in-the-api? If not, please create a PT story and PR for that.

Comment (karimkawambwa) [reply takwimu/takwimu_ui/src/components/DataContainer/EntitiesDataContainer.js]: The fetch is returning Ids.
I had tried the API Field but It did not seem to work.
I'll create a PT story might need to explore slightly more than I did.

**Pull request merged**